src/test/scala/nl.knaw.dans.easy.deposit/docs/DatasetMetadataSpec.scala

```diff
@@ -162,4 +163,113 @@ class DatasetMetadataSpec extends TestSupportFixture {
+        |		}
+        |	]
+        |}""".stripMargin
+    DatasetMetadata(dates) should matchPattern {
+      case Failure(ide: InvalidDocumentException) if ide.getMessage == """invalid DatasetMetadata: don't recognize {"dates":{"invalidOne":"invalid","invalidValue":"2018-05-31","invalidQualifier":"dcterms:created"}}""" =>
+    }
+  }
+
+  "DatasetMetadata.creators" should "only report the part of the creators that are wrong" in {
+    val creatorValid = """{"titles": "Msc", "initials": "H.A.M.", "surname": "Boter", "ids": [ { "scheme": "DAI", "value":  "93313935x"}, {"scheme": "BSN", "value": "1234"} ], "organization" :"DANS"}"""
+    val creatorInvalidIdElement = """{"titles": "Msc", "initials": "B.A.M.", "surname": "Hoter", "ids": [ { "scheme": "DAI", "value":  "93313935Z"}, {"scheme": "BSN", "value": "1235", "organization": "overheid"} ], "organization" :"DANS"}"""
+    val creatorInvalidElementAtRootLevel = """{"titles": "Aartshertog", "FirstName": "jan-willem-hendrik", "surname": "Oranje", "ids": [ { "scheme": "DAI", "value":  "93313935y"}, {"scheme": "BSN", "value": "9999"} ], "organization" :"DANS"}"""
+    //TODO moet hij hier niet struikelen over het veld role[0].waarde?
+    val creatorInvalidRole =
+      """{"titles": "Dr", "initials": "S.", "surname": "Pieterzoon", "ids": [ { "scheme": "DAI", "value":  "93313935i"} ], "organization" :"DANS", "role": [ { "scheme": "datacite:contributorType", "key": "ContactPerson", "waarde": "invalid", "andereWaarde": "invalid"} ] }"""
+    val creators = s"""{ "creators": [$creatorValid, $creatorInvalidIdElement, $creatorInvalidElementAtRootLevel, $creatorInvalidRole] }"""
+    DatasetMetadata(creators) should matchPattern {
+      case Failure(ide: InvalidDocumentException) if ide.getMessage == """invalid DatasetMetadata: don't recognize {"creators":[{"ids":{"organization":"overheid"}},{"FirstName":"jan-willem-hendrik"}]}""" =>
+    }
+  }
+
+  "DatasetMetadata.[creators, contributors]" should "only report the part of the creators and contributors that are wrong" in {
+    val metaData =
+      s"""{
+         |	"contributors": [{
+         |			"organization": "rightsHolder1",
+         |			"role": {
+         |				"scheme": "datacite:contributorType",
+         |				"key": "RightsHolder",
+         |				"value": "rightsholder"
+         |			},
+         |			"ids": [{
+         |				"scheme": "aScheme",
+         |				"key": "aKey",
+         |				"value": "aValue",
+         |				"waarde": "invalidProperty"
+         |			}]
+         |		},
+         |		{
+         |			"titles": "Dr.",
+         |			"initials": "A.S.",
+         |			"insertions": "van",
+         |			"surname": "Terix",
+         |			"role": {
+         |				"scheme": "datacite:contributorType",
+         |				"key": "RightsHolder",
+         |				"value": "rightsholder",
+         |				"otherKey": "placeHolder",
+         |				"otherValue": "placeHolder"
+         |			}
+         |		}
+         |	]
+         |}""".stripMargin
```

Comment:
Mix single/multiline layout. Some old examples could use the same style change. For example in this case:
```
    s"""{
       | "contributors": [{
       |   "organization": "rightsHolder1",
       |   "role": { "scheme": "datacite:contributorType", "key": "RightsHolder", "value": "rightsholder" },
       |   "ids": [{ "scheme": "aScheme", "key": "aKey", "value": "aValue", "waarde": "invalidProperty" }]
       |  },
       |  {
       |   "titles": "Dr.", "initials": "A.S.", "insertions": "van", "surname": "Terix",
       |   "role": { "scheme": "datacite:contributorType", "key": "RightsHolder", "value": "rightsholder", "otherKey": "placeHolder", "otherValue": "placeHolder" }
       |  }
       | ]
       |}""".stripMargin
```